front/src/components/map/MapAssistantButtons.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div class="assistant-buttons">
+    <v-btn
+      small
+      fab
+      :retain-focus-on-click="false"
+      @click="setMapToCurrentLocation"
+    >
+      <v-icon>mdi-crosshairs-gps</v-icon>
+    </v-btn>
+    <v-btn small fab class="mt-2" @click="refreshPosts">
+      <v-icon>mdi-refresh</v-icon>
+    </v-btn>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "MapAssistantButtons",
+  methods: {
+    async setMapToCurrentLocation() {
+      await this.$kakaoMap.setCenterByCurrentLocation();
```

Comment:
```suggestion
      await this.$kakaoMap
        .setCenterByCurrentLocation()
        .catch(() =>
          this.$store.commit(
            "snackbar/SHOW",
            ERROR_MESSAGE.UNIDENTIFIABLE_LOCATION,
          ),
        );
```

위치 정보를 받을 수 없을 때 로직도 필요할 것 같네요!